### ISSUE TITLE
Example images are included in npm repo

### ISSUE THREAD

**Issue by josephg** [jossmac/react-images#40]
```
$ du -hs node_modules/react-images/*
...
6.5M	node_modules/react-images/examples
```

Probably don't need to download 6 megs of cute animal photos on every deploy. Add a `.npmignore` file to remove the examples folder. [docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package).

**Comment by b2whats** [comment]
Why? Traffic ?

**Comment by josephg** [comment]
Yep. Npm is slow enough as it is.

**Comment by jossmac** [comment]
Cheers @josephg 

I intend to replace with Unsplash images from API, see #31

**Comment by josephg** [comment]
Cool. Removing the images from the npm package would also work fine.

**Comment by englercj** [comment]
An actual problem this causes in on small embedded devices where disk space is limited. For example, the Onion Omega has 7.5M of storage for user-space. Right now I am *unable to install node.js* because npm takes ~6.4M of space on the device.

Until this change is made I am unable to install or use npm on this device.

**Comment by GregoryPotdevin** [comment]
This should be fixed with the latest commits once they get published to npm.

**Comment by jossmac** [comment]
Resolved by #78
